pyproject.toml

```diff
@@ -140,7 +140,7 @@ inputs = "*"
 Jinja2 = "*"
 lru-dict = "*"
 matplotlib = "*"
-metadrive-simulator = { version = "0.4.2.3", markers = "platform_machine != 'aarch64'" } # no linux/aarch64 wheels for certain dependencies
+metadrive-simulator = {git = "https://github.com/metadriverse/metadrive.git", rev ="233a3a1698be7038ec3dd050ca10b547b4b3324c", markers = "platform_machine != 'aarch64'" } # no linux/aarch64 wheels for certain dependencies
```

Comment:
can we just unpin it? have they done a release since that fix? I don't remember why we pinned it in the first place

Reply:
Unpinned. 0.4.2.3 is the last they did and that was before the fix

Reply:
oh then a git commit is fine if there's no release. just leave a comment to explain why so we can change it later